sources/packages/web/src/views/student/StudentDashboard.vue

```diff
@@ -16,22 +19,31 @@ import { useStore } from "vuex";
 import { StudentService } from "@/services/StudentService";
 import formio from "@/components/generic/formio.vue";
 import RestrictionBanner from "@/views/student/RestrictionBanner.vue";
-import FullPageContainer from "@/components/layouts/FullPageContainer.vue";
 import CheckValidSINBanner from "@/views/student/CheckValidSINBanner.vue";
+import { useRouter } from "vue-router";
+import { useToastMessage } from "@/composables";
 
 export default {
+  emits: ["goToStudentApplication"],
```

Comment:
Emits is used when a component is emitting a event to a consumer. Please remove it.